validator/sawtooth_validator/server/core.py

```diff
@@ -187,7 +188,7 @@ def __init__(self, network_endpoint, component_endpoint, peer_list,
         self._journal = Journal(
             consensus_module=dev_mode_consensus,
```

Comment:
The journal and journal components have not been updated to use the on chain configuration.